README.md

```diff
@@ -63,22 +63,35 @@ You can get your dev environment up and running with installing only Docker and
 
 ## Running the docker stack
 
-In the project's root directory, run `make dev_up`. This builds the dev version of the docker images and launches the containers using docker-compose. Stop the running stack with `make dev_down`.
+In the project's root directory, run `make new`. This pulls the dev-tagged "expungeservice" image and launches the containers using docker-compose. Stop the running stack with `make stop`.
 
-After running `make dev_up`, you can navigate to `localhost` in the browser and see the frontend running.  You can now log in using either of the following credentials
+After this target completes, you can navigate to [http://localhost:3000](http://localhost:3000) in the browser and connect to the React dev server with full hot-module reloading. This may take a minute or two to come up before it is available. Check the service with `make frontend_logs`. Once everything is up, you can log in using either of the following credentials:
 
 * Email: admin@email.com, Password: admin
 * Email: user@email.com, Password: user
 
-If you run `docker ps`, you can see the frontend app running on `localhost:3000`, however if you try logging into this instance instead of the frontend at `localhost`, you will get a 500 server error.
+In the course of backend development, one may not need to be running the React/HMR dev server. To build the frontend static files, and use the backend to serve them, run the following command:
 
-Whenever a dependency is added to the frontend or backend, you will need to rebuild the docker images or you will get errors when trying to run the stack. To do so, take down the stack, rebuild, and restart stack with:
 ```
-make dev_down
-make dev_build
-make dev_up
+$ make frontend_down frontend_build
 ```
 
+Then navigate to [http://localhost:5000](http://localhost:5000) to access the backend service directly. This is configured to serve the static files. `make up` will start the HMR dev server again at 3000 if you need it.
```

Comment:
This is a bit confusing, and needs some context: the frontend uses `react-scripts` which means when you run `npm run start`, it launches a "hot-module reloading" dev server that also proxies the backend. This is very useful for frontend development, but comes at the cost of running a "watcher" process in the container for rebuilding/hot-reloading when files change. This can be unwanted overhead when only doing backend development. The flask app has added static file code now, which serves from the `src/frontend/build` directory, exactly where react-scripts builds static files. SO - for frontend dev and hot module reloading, make sure the node service is running and use 3000. For backend dev only, stop the node service, build the frontend static files, and use 5000. Ha, not sure if I cleared anything up here.